code/Example.java

```diff
@@ -1,5 +1,5 @@
 public class Example {
-
+  // Test the out of office.
   public static void main(String[] args) {
```

Comment:
Hey team, I am going to be OOO the rest of the week.
/ooo tomorrow until next week